dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -68,8 +68,8 @@ include "dpnp_algo_statistics.pyx"
 include "dpnp_algo_trigonometric.pyx"
 
 
-ctypedef void(*fptr_dpnp_arange_t)(size_t, size_t, void * , size_t)
-ctypedef void(*fptr_dpnp_initval_t)(void * , void * , size_t)
+ctypedef void(*fptr_dpnp_arange_t)(size_t, size_t, void *, size_t)
+ctypedef void(*fptr_dpnp_initval_t)(void *, void * , size_t)
```

Comment:
Pay attention to spaces at ` void * `. Here our code style tool can't decide should we have a space after star or not, and it happens in the same line and same definition.

Reply:
Seems like codestyle tool doesn't work correct for Cython files.

Reply:
This is one of the thing we need to improve